.github/workflows/lint.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           git diff --exit-code
           exit "${EXIT_CODE?}"
 
-  yapf:
+  black:
```

Comment:
Can you add a link to https://black.readthedocs.io/en/stable/ and/or https://pypi.org/project/black/ in the PR description?

Reply:
Good idea, done